app/controllers/api/feedback_submissions_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  class FeedbackSubmissionsController < ApiController
+    def create
+      @feedback = FeedbackSubmission.new(feedback_params)
+
+      if @feedback.save
+        ZendeskTicketsJob.perform_later(@feedback)
+      else
+        render status: 422, json: { messages: @feedback.errors.full_messages }
```

Comment:
@alan I've been loosely following the aspects I like from Github's API, in which there is a message and then an array of errors. See https://developer.github.com/v3/#client-errors.

So in this case the error would be something like "Validation Failed" and then there would be a list of errors.

_However_, I would prefer to do validation on the client side (i.e. in public) rather than using the API for this. This is the pattern for the booking process, in which case creating a booking is expected to succeed and failure is an error state. I see no reason to do differently for the API.